extensions/cpp/package.json

```diff
@@ -50,6 +50,13 @@
           "cpp"
         ],
         "configuration": "./language-configuration.json"
+      },
+      {
+        "id": "cuda-cpp",
+        "extensions": [
+          ".cu",
+          ".cuh"
+        ]
       }
     ],
     "grammars": [
```

Comment:
Is there any value to having a cuda-cpp grammar here that just points to the existing C++ grammar?

Reply:
Hi Alex, we have 2 PRs (one for adding a lang id and one for updating build.js to pull down the grammar and lang-configuration for cuda-cpp (and adding the new grammar to package.json)). But, the second PR is coming out with a delay due to the lack of availability of the maintainer for better-cpp. That being said, we'd be happy to combine the 2 PRs if that makes more sense.

Reply:
By the way, @alexr00, is there an issue at the top of the branch? I can't get main to build anymore (yarn terminates with an error when trying to delete a file using `rmdir`)